docs/other/networking.md

```diff
@@ -1,12 +1,15 @@
-Since Vagrant manages internal network connections for Drupal VM, and since every person's network setup is unique, there are sometimes networking issues that require some adjustments to your Drupal VM configuration or a computer reboot.
+## DHCP (default)
 
-## Network Route Collision
+By default Drupal VM is assigned an IP automatically via DHCP. This is configured when `vagrant_ip` set to `dhcp` in `default.config.yml` or your `config.yml`.
+
+## Static IP
 
-Drupal VM comes configured with the default IP address `192.168.88.88`. If you're connected to a LAN with the same private IP address range (e.g. `192.168.x.x`), then VirtualBox or VMware will not be able to set up the VM with the default IP address, because that would conflict with the `192.168.x.x` network your computer is using.
+To use a static IP, change the `vagrant_ip` in `config.yml` to the desires private IP address.
 
-In this case, you have two options:
+## `vagrant-auto_network` plugin
 
-  1. Switch the `vagrant_ip` in `config.yml` to a different private IP address, e.g. `172.16.0.88` or `10.0.1.88`.
-  2. Install the `vagrant-auto_network` plugin (`vagrant plugin install vagrant-auto_network`), and set the `vagrant_ip` to `0.0.0.0`.
+If you prefer using the `vagrant-auto_network` plugin (`vagrant plugin install vagrant-auto_network`) to assign your IP, set the `vagrant_ip` to `0.0.0.0`.
+
+## Network Route Collision
 
-Another cause of route collisions is the use of multiple VM providers on your computer. If you have both VirtualBox and VMware Fusion, and you have VMs running in both, and you attempt to use the same IP range in both providers, you'll hit a networking conflict. In this case, the only easy way to restore connectivity is to restart your host machine.
+If you have multiple VM providers running VMs on your computer, and you attempt to use the same IP range in both providers, you'll hit a networking conflict. In this case, the only easy way to restore connectivity is to restart your host machine.
```

Comment:
I think this can be removed. I got this when I just attempted to launch two providers:

> An active machine was found with a different provider. Vagrant
currently allows each machine to be brought up with only a single
provider at a time. A future version will remove this limitation.
Until then, please destroy the existing machine to up with a new
provider.

Reply:
Maybe skip the entire file?